modules/farm_fd2/src/entrypoints/direct_seeding/direct_seeding.submitReset.e2e.cy.js

```diff
@@ -242,24 +242,29 @@ describe('Direct Seeding: Submit/Reset Buttons component', () => {
       .find('[data-cy="selector-input"]')
       .should('have.value', '1');
 
-    cy.get('[data-cy="multi-equipment-selector"]')
-      .find('[data-cy="selector-1"]')
-      .find('[data-cy="selector-input"]')
-      .should('have.value', null);
+    cy.get('[data-cy="multi-equipment-selector"]').should('not.be.visible');
     cy.get('[data-cy="soil-disturbance-depth"]').should('not.exist');
     cy.get('[data-cy="soil-disturbance-speed"]').should('not.exist');
     cy.get('[data-cy="soil-disturbance-area"]').should('not.exist');
 
+    cy.get(
+      '[data-cy="direct-seeding-soil-disturbance-accordion-title"]'
+    ).click();
+    cy.get('[data-cy="multi-equipment-selector"]').should('be.visible');
+
     cy.get('[data-cy="multi-equipment-selector"]')
```

Comment:
There should be a check before this to ensure that the value of the `selector-input` is `null` after the reset and before "Tractor" is selected.